packages/kit/src/views/Setting/pages/Notifications/NotificationsSettings.tsx

```
@@ -26,8 +26,13 @@ import { ETranslations } from '@onekeyhq/shared/src/locale';
 import platformEnv from '@onekeyhq/shared/src/platformEnv';
 import { EModalRoutes, EModalSettingRoutes } from '@onekeyhq/shared/src/routes';
 import { EModalNotificationsRoutes } from '@onekeyhq/shared/src/routes/notifications';
-import type { INotificationPushSettings } from '@onekeyhq/shared/types/notification';
+import timerUtils from '@onekeyhq/shared/src/utils/timerUtils';
+import {
+  ENotificationPermission,
+  type INotificationPushSettings,
+} from '@onekeyhq/shared/types/notification';
 
+import { MultipleClickStack } from '../../../../components/MultipleClickStack';
```

ESLint

[error] Too many parent references (4 while maximum of 3 allowed)

(import-path/parent-depth)